openpype/hosts/maya/plugins/publish/collect_shots.py

```diff
@@ -0,0 +1,148 @@
+                "frameEnd": frame_end,
+                "clipIn": frame_start,
+                "clipOut": frame_end,
+                "fps": mel.eval("currentTimeUnitToFPS()"),
+                "resolutionWidth": asset_entity.get(
+                    width_key, project_entity.get(width_key, 1920)
+                ),
+                "resolutionHeight": asset_entity.get(
+                    height_key, project_entity.get(height_key, 1080)
+                ),
+                "pixelAspect": asset_entity.get(
+                    pixelAspect_key, project_entity.get(pixelAspect_key, 1)
+                ),
+                "asset": instance.data["name"],
+                # Disabling task fetching in collect_anatomy_instance_data
+                "task": None,
+                # For extract_maya_scene_raw.
+                "exactSetMembersOnly": False
+            }
+        )
+
+        # Set hierarchy parents.
+        instance.data["parents"] = []
+        context_parents = instance.context.data.get("parents")
+        if context_parents is None:
+            parents = get_parents(project_entity["name"], asset_entity)
+            for parent in reversed(parents):
+                instance.data["parents"].append(
+                    {
+                        "entity_name": parent["name"],
+                        "entity_type": parent["type"]
+                    }
+                )
+            instance.data["parents"].append(
+                {
+                    "entity_name": asset_entity["name"],
+                    "entity_type": asset_entity["type"]
+                }
+            )
+            instance.context.data["parents"] = instance.data["parents"]
+        else:
+            instance.data["parents"] = instance.context.data["parents"]
+
+        self.log.debug(
+            json.dumps(instance.data["parents"], indent=4, sort_keys=True)
+        )
+
+        # Set frame ranges.
+        attributes = instance.data["creator_attributes"]
+        if attributes["use_start_frame"]:
+            instance.data["frameOffset"] = (
+                attributes["start_frame"] - frame_start
+            )
+            instance.data["frameStart"] = attributes["start_frame"]
+            instance.data["frameEnd"] = (
+                (frame_end - frame_start) + attributes["start_frame"]
+            )
+
+        if attributes["use_handles"]:
+            instance.data["handleStart"] = attributes["handle_start"]
+            instance.data["handleEnd"] = attributes["handle_end"]
+
+        instance.data["update_timeline"] = attributes["update_timeline"]
+
+
+class CollectShotsData(pyblish.api.ContextPlugin):
+    """Collect shots data."""
+
+    order = CollectShotData.order + 0.01
+    label = "Collect Shots Data"
+    families = ["shot"]
+
+    def process(self, context):
+        # Collect everything from the scene to export, except shot instances.
+        nodes_to_export = get_highest_in_hierarchy(cmds.ls(dagObjects=True))
```

Comment:
```suggestion
        nodes_to_export = cmds.ls(assemblies=True)
```
Pretty sure this would be equivalent?